backend/api/migrations/0001_initial.py

```diff
@@ -0,0 +1,118 @@
+                ('user_permissions', models.ManyToManyField(blank=True, help_text='Specific permissions for this user.', related_name='user_set', related_query_name='user', to='auth.permission', verbose_name='user permissions')),
+            ],
+            options={
+                'db_table': 'users',
+                'managed': True,
+            },
+            managers=[
+                ('objects', django.contrib.auth.models.UserManager()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='MedicationInfo',
+            fields=[
+                ('medication_id', models.AutoField(primary_key=True, serialize=False)),
+                ('dosage', models.CharField(max_length=8)),
+                ('medication_time', models.JSONField()),
+                ('misc_information', models.CharField(blank=True, max_length=200, null=True)),
+                ('frequency', models.CharField(blank=True, max_length=20, null=True)),
+                ('medication_name', models.CharField(blank=True, max_length=30, null=True)),
+                ('active_medication', models.BooleanField(blank=True, null=True)),
+                ('start_date', models.DateField(blank=True, null=True)),
+                ('end_date', models.DateField(blank=True, null=True)),
+                ('user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'db_table': 'medication_info',
+                'managed': True,
+            },
+        ),
+        migrations.CreateModel(
+            name='Refills',
+            fields=[
+                ('refill_id', models.AutoField(primary_key=True, serialize=False)),
+                ('active', models.BooleanField(blank=True, null=True)),
+                ('refill_date', models.CharField(blank=True, max_length=10, null=True)),
+                ('refill_time', models.TimeField(blank=True, null=True)),
+                ('num_meds', models.IntegerField(blank=True, null=True)),
+                ('medication', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='api.medicationinfo')),
+                ('user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'db_table': 'refills',
+                'managed': True,
+            },
+        ),
+        migrations.CreateModel(
+            name='SymptomLog',
+            fields=[
+                ('symptom_id', models.AutoField(primary_key=True, serialize=False)),
+                ('date', models.DateField()),
+                ('time', models.TimeField()),
+                ('description', models.CharField(max_length=200)),
```

Comment:
Why are we limiting user descriptions to 200 characters?

Reply:
Same as above